lib/infrastructure/auth/firebase_auth_repository.dart

```diff
@@ -111,10 +111,12 @@ class FirebaseAuthRepository implements IAuthRepository, Disposable {
 
   @override
   Future<Either<AuthFailure, Unit>> updateUsername({
```

Comment:
This method should be removed, it's the responsibility of the ProfileRepository

Reply:
Done